src/index.html

```diff
@@ -1,13 +1,426 @@
+          </div>
+        </div>
+      </section>
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desctop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+
+                <div class="contact-us__info-link">
+                  <a
+                    href="tel:+1 234 5555-55-55"
+                    class="contact-us__info-link"
+                  >
+                    +1 234 5555-55-55
+                  </a>
+                </div>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+
+                <div class="contact-us__info-link">
+                  <a
+                    href="mailto:hello@miami.com"
+                    class="contact-us__info-link"
+                  >
+                    hello@miami.com
+                  </a>
+                </div>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info  contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Address
+                </div>
+
+                <div class="contact-us__info-link">
+                  <a
+                    href="https://goo.gl/maps/KM4DWEbuQdzhG7Bw7"
+                    class="contact-us__info-link"
+                  >
+                    400 first ave. n.
+                    <br>
+                    suite 700
+                    <br>
+                    Minneapolis, MN
+                    <br>
+                    55401
+                  </a>
+                </div>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--8-12">
+              <form action="#" method="get">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contact-us__field form-field"
+                  id="n"
+                >
+                <label for="n"></label>
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contact-us__field form-field"
+                  id="e"
+                >
+                <label for="e"></label>
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                  contact-us__field
+                  contact-us__field--last
+                  form-field 
+                  form-field--textarea"
+                >
+                </textarea>
```

Comment:
```suggestion
                  form-field--textarea"
               ></textarea>
```

Зараз існує баг, коли плейсхолдер в textarea не відображається. Щоб це пофіксити, потрібно використати таке форматування, як я навів у прикладі (ОДИН відступ вліво і закриваючий тег на тому ж рядку, що і відкриваючий)

<img width="548" alt="Снимок экрана 2021-12-21 в 17 48 55" src="https://user-images.githubusercontent.com/66570142/146959381-d076d473-9eed-4a2f-a510-651aec8e7b43.png">